oak-lucene/src/test/java/org/apache/jackrabbit/oak/plugins/index/lucene/hybrid/LuceneJournalPropertyBuilderTest.java

```diff
@@ -53,7 +53,25 @@ public void nullOrEmptyJson() throws Exception{
 
         assertTrue(Iterables.isEmpty(((IndexedPaths)builder2.build())));
     }
+    @Test
+    public void addJsonLessThanMaxBuilderSize() throws Exception {
+        String a = null;
+        for (int i = 0; i < 499; i++) {
+            a = "{\"/var/eventing/jobs/foo/2022/4/19/14/27/af96fcfa9e32_8589" + i + "\" :[\"/oak:index/foo\",\"/oak:index/bar\"]}";
+            builder.addSerializedProperty(a);
+        }
+        assertEquals(createdIndexPathMap((IndexedPaths)builder.build()).size(), 998);
```

Comment:
Please swap the two arguments for assertEquals(). The first argument is the expected value, which is 998 in this test.